tests/benchmarks/sys_kernel/testcase.yaml

```diff
@@ -6,4 +6,4 @@ tests:
   benchmark.kernel.qemu_cortex_m3:
     platform_whitelist: qemu_cortex_m3
     extra_args: CONF_FILE=prj_qemu_cortex_m3.conf
-    tags: benchmark qemu_cortex_m3
\ No newline at end of file
+    tags: benchmark qemu_cortex_m3
```

Comment:
remove this change to this file, not sure why we have qemu_cortex_m3 in the tags